caddy/Caddyfile

```diff
@@ -0,0 +1,13 @@
+{
+    order vulcain before request_header
```

Comment:
The other option is you could have users use a `route` to wrap vulcain + reverse_proxy (in that order).